nbclient/tests/test_client.py

```diff
@@ -345,11 +345,7 @@ def test_async_parallel_notebooks(capfd, tmpdir):
     res = notebook_resources()
 
     with modified_env({"NBEXECUTE_TEST_PARALLEL_TMPDIR": str(tmpdir)}):
-        tasks = [
-            async_run_notebook(input_file.format(label=label), opts, res) for label in ("A", "B")
-        ]
-        loop = asyncio.get_event_loop()
-        loop.run_until_complete(asyncio.gather(*tasks))
+        [async_run_notebook(input_file.format(label=label), opts, res) for label in ("A", "B")]
```

Comment:
You removed the execution of these tasks, can you explain why?

Reply:
I think this was a mistake on my part when I was trying to investigate some failures. I've added those changes back in.